src/backend/common/helpers/tests/event_helper_test.py

```diff
@@ -240,10 +278,14 @@ def test_sort_events(ndb_context) -> None:
     EventHelper.sort_events(events)
     assert events == [e2, e1]
 
+    events = [e2, e1]
```

Comment:
Note to future people who are like "why would we ever do this" - mostly just to make sure that this method is sorting events.  Passing the array in different states to ensure it's always the same does this. For a larger array you might want to pass the array in a random order during every test run but - this is just two elements, so this is easier